src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -234,11 +243,7 @@ class CosmasGoogleCloudService(private val bucketName: String,
             handleStorageException(e, responseObserver)
             return
         }
-        val newTomb = CosmasProto.FileTomb.newBuilder().
-                setFileId(request.fileId).
-                setFileName(request.fileName).
-                setRemovalTimestamp(request.removalTimestamp).
-                build()
+        val newTomb = CosmasProto.FileTomb.newBuilder().setFileId(request.fileId).setFileName(request.fileName).setRemovalTimestamp(request.removalTimestamp).build()
```

Comment:
Please revert to formatting with linebreaks before every setter call, it is way more readable. This applies through the whole pull request.

Reply:
I hope I fixed it everywhere